src/commands/generate-application/index.ts

```diff
@@ -15,7 +15,8 @@ export enum Platforms {
 }
 
 export enum UseCasesAppNames {
-  portableReputation = 'portable-reputation',
+  gaming = 'gaming',
+  career = 'career',
```

Comment:
Other use cases have "ReferenceApp" suffix. Let's be consistent with them (even tho it's a bit ugly).

Reply:
In other places it's changed, but this file is not changed 🤔